### ISSUE TITLE
Tetrix 3d model

### ISSUE THREAD

**Issue by yangwao** [kodadot/metaprime-landing#1]
This Tetrix will be replaced with [model-viewer](https://modelviewer.dev/) if possible using this model https://sketchfab.com/3d-models/sierpinski-triangle-6535bb3393b14973bc84a35c28cd886c

Feel for end user on mobile would be use gyro to rotate automatically with model :)

IP rights are solved, we've got permission from creator to use it, paying him coffee ;)

![image](https://user-images.githubusercontent.com/5887929/137619905-030439df-ea7f-4290-9fd3-d2dc31dee354.png)

**Comment by yangwao** [comment]
Temporary use this, but dunno how to get alpha from black bg?
![image](https://user-images.githubusercontent.com/5887929/137620652-670ed166-89ce-4761-a9a1-c7ecbd0eb707.png)

**Comment by JKrupinski** [comment]
@yangwao  showing 3d model view will decrease performance significantly especially when you want to add it in main hero.
Please rethink if you want to use it, if so I think we should show it in other section e.g AUGMENTED BONDING
CURVE WITH A FOCUS
ON PUBLIC GOODS
and try to lazyload it maybe

**Comment by yangwao** [comment]
yeah probably we could post pone it to v1.1 version till we figure out stuff, maybe we can have css3d version which should not be that bad for mobile etc

**Comment by yangwao** [comment]
Found other candidats till then https://codepen.io/KennyLindahl/pen/lABqD
http://stemkoski.github.io/Three.js/CSS3D.html
https://codepen.io/drake-p/pen/bhICx -- make it incremental and revert? 
https://codepen.io/MaxXiong/pen/svqGn?js-preprocessor=babel

good one https://www.reddit.com/r/blackmagicfuckery/comments/nvnvmh/chaos_black_magic/

okay we can find something later in day, no worries!

**Comment by yangwao** [comment]
Found at https://v3.nuxtjs.org/

Can we use something like this? 

https://user-images.githubusercontent.com/5887929/141777398-2f1c9e42-8df0-4d7b-a83f-f0f8694c7ded.mov

**Comment by yangwao** [comment]
https://theconvo.space/

![image](https://user-images.githubusercontent.com/5887929/142391299-074c12d6-e06f-4740-bc64-69281efe2a78.png)
